### PR TITLE
fix #80061 tekno.kompas.com

### LINKED ISSUE #80061: tekno.kompas.com
### Issue URL (Annoyance)
 [https://tekno.kompas.com/read/2021/03/23/08022177/poco-x3-pro-resmi-meluncur-ini-harganya](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftekno.kompas.com%2Fread%2F2021%2F03%2F23%2F08022177%2Fpoco-x3-pro-resmi-meluncur-ini-harganya)
### Comment
Username: @3xploiton3
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Firefox
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Block WebRTC, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdBlockID

### PR DESCRIPTION
**Pull request by stanislav-atr**
#80061

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -118,6 +118,7 @@ revouninstaller.com#$##popupModal { display: none !important; }
 revouninstaller.com#$#.modal-backdrop { display: none !important; }
 revouninstaller.com#$#body { overflow: visible !important; padding-right: 0 !important; }
 techmeme.com###nlsb
+kompas.com##.root > div.dark-fixed
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:121]: I'm not sure that we should hide this element, this is part of site's functionality

File: MobileFilter/sections/specific_web.txt [hunk 2]
```diff
@@ -69,6 +69,7 @@ n-tv.de##.topbanner-mobile
 syosetu.org##.ad
 m.bloomberght.com##.zone-area
 ezgif.com###hdr
+kompas.com###js-banner-sticky
```
Comment (Sergey-Lyapin) [inline MobileFilter/sections/specific_web.txt:72]: I don't see this element

Comment (stanislav-atr) [reply MobileFilter/sections/specific_web.txt:72]: <details><summary>Screenshot:</summary>

image</details><br/>

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/popups.txt:121]: I thought so too, but it covers the content and is really annoying.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:121]: @Alex-302 decide =)

File: MobileFilter/sections/specific_web.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply MobileFilter/sections/specific_web.txt:72]: `kompas.com##.boxBanner1` from AdBlockID hides this element

**Pull request merged**